RiotSwiftUI/Modules/UserSessions/Common/DeviceType.swift

```diff
@@ -38,17 +38,15 @@ enum DeviceType {
     }
     
     var name: String {
-        let appName = AppInfo.current.displayName
```

Comment:
@pixlwave sorry just pinging to inform you that I've decided to include this in the current PR as it's related. We shouldn't use the _current_ app name here because the device type name is used for _all_ sessions, so there's no guarantee the others are Element, too.